pkg/api/addons.go

```diff
@@ -17,16 +17,24 @@ func (cs *ContainerService) setAddonsConfig(isUpdate bool) {
 	k8sComponents := K8sComponentsByVersionMap[o.OrchestratorVersion]
 	specConfig := cloudSpecConfig.KubernetesSpecConfig
 	defaultsHeapsterAddonsConfig := KubernetesAddon{
-		Name:    DefaultHeapsterAddonName,
-		Enabled: to.BoolPtr(DefaultHeapsterAddonEnabled),
+		Name:    HeapsterAddonName,
+		Enabled: to.BoolPtr(DefaultHeapsterAddonEnabled && !common.IsKubernetesVersionGe(o.OrchestratorVersion, "1.13.0")),
 		Containers: []KubernetesContainerSpec{
 			{
-				Name:  DefaultHeapsterAddonName,
-				Image: specConfig.KubernetesImageBase + k8sComponents["heapster"],
+				Name:           HeapsterAddonName,
+				Image:          specConfig.KubernetesImageBase + k8sComponents["heapster"],
+				CPURequests:    "88m",
```

Comment:
Moving the resource configuration definitions for this addon into this default assignment flow, out of the static definition in the `yaml`.